### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_UsingVersionControlinVSCode_ScreenReader:Focus order improper when we navigate to heading while screen reader is on.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3587]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.3.1;#MAS2.4.3;#Win10;#Visual Studio Code;#CELAP1_code;#Chrome;#NVDA;#Narrator

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/editor/versioncontrol
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)
Chrome Version:Version 81.0.4044.113 (Official Build) (64-bit)
Screen Reader: Narrator(Desktop App), NVDA(2019.3.1)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/docs/editor/versioncontrol
2)Using Version Control in VS Code Page will get open.
3)Navigate till the heading "Using Version Control in VS Code" while narrator is on.

**Actual:**
While we navigate from left side navigation panel to the heading "Using Version Control in VS Code" when narrator is on using caps+right arrow key the focus is first sifted to the "edit" button present on the page and then to the heading.

**Same issue has been observed in Chrome+NVDA also.**

**Same issue has been observed in following pages:**
1.https://code.visualstudio.com/docs/editor/whyvscode
2.https://code.visualstudio.com/docs/getstarted/settings
3.https://code.visualstudio.com/docs/python/python-tutorial
4.https://code.visualstudio.com/docs/getstarted/tips-and-tricks
5.https://code.visualstudio.com/docs/setup/linux
6.https://code.visualstudio.com/docs/editor/integrated-terminal

**Expected:**
While we navigate using caps+right arrow key to the content on the page the focus should be first shifted to the heading and then to the "edit" button that is present on the screen.

**User Impact:**
Users might get confused as they might not know what content are they going to edit as the "edit" button directly receives the first keyboard focus and then the heading is focused.

**Attachment link for Reference:**
[3587_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_UsingVersionControlinVSCode_ScreenReader.zip](https://github.com/microsoft/vscode-docs/files/4510977/3587_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_UsingVersionControlinVSCode_ScreenReader.zip)

**Comment by gregvanl** [comment]
Won't fix. Same behavior as docs.microsoft.com.

**Comment by ManishaC1** [comment]
As per the internal discussion, this is not an issue. 
Github tags:#Closed
